chia/server/start_service.py

```diff
@@ -190,11 +188,13 @@ async def start(self) -> None:
                 self.upnp.remap(port)
 
         await self._server.start(
-            listen=self._listen,
             prefer_ipv6=self.config.get("prefer_ipv6", False),
             on_connect=self._on_connect_callback,
         )
-        self._advertised_port = self._server.get_port()
+        try:
+            self._advertised_port = self._server.get_port()
+        except ValueError:
+            pass
```

Comment:
in this case, we'll just leave the class not having the `_advertised_port` field at all? I would have expected we'd want to at least set it to `None`

Reply:
It looks like it is set unconditionally in `.__init__()`.  I think this is just handling the case where we had requested `0` and now we ought to be filling in the actual port we opened.  I think...

Reply:
Ping me if it is decided that no action is needed on this